lib/sles4sap/azure_cli.pm

```diff
@@ -1217,3 +1219,72 @@ sub az_network_peering_delete {
     assert_script_run($az_cmd);
 }
 
+=head2 az_disk_create
+
+    az_disk_create(resource_group=>$resource_group, name=>$name
+        [, size_gb=>60, source=$source, tags="tag1=value1 tag2=value2"]);
+
+B<resource_group> Existing resource group name.
+
+B<source> Create disk by cloning snapshot
+
+B<size_gb> New disk size
+
+B<name> New disk name
+
+B<tags> Additional tags to add to the disk resource. key=value pairs must be separated by empty space.
+    Example: az_disk_create(tags=>"some_tag=some_value another_tag=another_value")
+
+Creates new disk device either by specifying B<size_gb> or by cloning another disk device using argument B<source>.
```

Comment:
Usually we have comment with these section in this order:
1. example
2. description
3. arguments

This one has order like 1,3,2 instead

Reply:
maybe move `name` just below `resource_group` as these two are mandatory

Reply:
Oh yes... you are right.
Fixed and updated.